dev/io.openliberty.microprofile.openapi.2.0.internal_fat/fat/src/io/openliberty/microprofile/openapi20/fat/deployments/MergeConfigServerXMLTest.java

```diff
@@ -63,18 +68,300 @@ public class MergeConfigServerXMLTest {
 
     @BeforeClass
     public static void startup() throws Exception {
+        server.saveServerConfiguration();
         server.startServer();
     }
 
     @AfterClass
     public static void shutdown() throws Exception {
-        server.stopServer("CWWKO1683W"); // Invalid info element
+        server.stopServer("CWWKO1683W", // Invalid info element
+                          "CWWKO1678W", // Invalid application name
+                          "CWWKO1679W" // Invalid module name
+        );
     }
 
     @After
     public void cleanup() throws Exception {
-        server.deleteAllDropinApplications();
-        server.removeAllInstalledAppsForValidation();
+        server.setMarkToEndOfLog();
+        server.restoreServerConfiguration(); // Will stop all apps deployed via server.xml and clear openapi config
+        server.waitForConfigUpdateInLogUsingMark(null);
+
+        server.deleteAllDropinApplications(); // Will stop all dropin apps
+        server.removeAllInstalledAppsForValidation(); // Validates that all apps stop
+
+        // Delete everything from the apps directory
+        server.deleteDirectoryFromLibertyServerRoot("apps");
+        LibertyFileManager.createRemoteFile(server.getMachine(), server.getServerRoot() + "/apps").mkdir();
+    }
+
+    @Test
+    public void testDefaultInclusion() throws Exception {
+        setMergeConfig(null, null, null);
+
+        // deploy app 1
+        WebArchive war1 = ShrinkWrap.create(WebArchive.class, "test1.war")
+                                    .addClasses(DeploymentTestApp.class, DeploymentTestResource.class);
+        deployApp(war1);
+
+        // deploy app 2
+        WebArchive war2 = ShrinkWrap.create(WebArchive.class, "test2.war")
+                                    .addClasses(DeploymentTestApp.class, DeploymentTestResource.class);
+        deployApp(war2);
+
+        // Download the documentation
+        String doc = OpenAPIConnection.openAPIDocsConnection(server, false).download();
+        JsonNode openapiNode = OpenAPITestUtil.readYamlTree(doc);
+
+        if (OpenAPITestUtil.getOpenAPIFeatureVersion() < 4.0f) {
+            // Default is first module only
+            // check that documentation includes only app 1
+            OpenAPITestUtil.checkPaths(openapiNode, 1, "/test");
+
+            // Test that a message was output
+            assertThat(server.findStringsInLogs(" I CWWKO1663I:.*Combining OpenAPI documentation from multiple modules is disabled.*test1"),
```

Comment:
As per a previous discussion about CWWKO1663I. We should remove test1 from here

Reply:
Done